File: modules/09-bad_seed/sunshinectf17_prepared/myexp.c
/*
 * @Author: Samrito
 * @Date: 2022-03-01 22:03:56
 * @LastEditors: Samrito
 * @LastEditTime: 2022-03-01 22:05:02
 */
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main() {
  time_t seed = time(NULL);
  srand(seed);
  for (int i = 0; i <= 49; ++i) {
    int num = rand() % 100;
    printf("%d\n", num);
  }
  return 0;
}

File: modules/12-z3/hs19_abyte/myexp.py
'''
Author: Samrito
Date: 2022-03-03 15:51:02
LastEditors: Samrito
LastEditTime: 2022-03-03 16:01:33
'''

s1 = b'irbugzv1v^x1t^jo1v^e5^v@2^9i3c@138|'

flag = []
for e in s1:
    flag.append(chr(e ^ 1))
print(''.join(flag))